src/store/actions/loadExpressionLanguages.ts

```diff
@@ -13,18 +13,27 @@ export interface Payload {
 }
 
 export default async ({ commit, getters }: Context): Promise<void> => {
-  const expressionLangs = getters.getAllExpressionLanguagesNotLoaded;
-  console.log({ expressionLangs });
-  for (const [, lang] of expressionLangs.entries()) {
-    const language = await getLanguage(lang);
-    console.log("Got language", language);
-    if (language !== null) {
-      const uiData: ExpressionUIIcons = {
-        languageAddress: language!.address!,
-        createIcon: language!.constructorIcon!.code!,
-        viewIcon: language!.iconFor!.code!,
-      };
-      commit("addExpressionUI", uiData);
+  try {
+    const expressionLangs = getters.getAllExpressionLanguagesNotLoaded;
+    console.log({ expressionLangs });
+    for (const [, lang] of expressionLangs.entries()) {
+      const language = await getLanguage(lang);
+      console.log("Got language", language);
+      if (language !== null) {
+        const uiData: ExpressionUIIcons = {
+          languageAddress: language!.address!,
+          createIcon: language!.constructorIcon!.code!,
+          viewIcon: language!.iconFor!.code!,
+        };
+        commit("addExpressionUI", uiData);
+      }
     }
+  } catch (e) {
+    commit("setToast", {
```

Comment:
I think it would be better to have a utility function to show & hide the toast, which can be used across the app, what do you think?

Reply:
If you look at `LeftDrawer.vue` at line `171`, we can set this from everywhere in the app, by committing a mutation. 

But maybe we should not use setToast inside actions, and only explicitly do it where we dispatch actions inside components? We could also create some mutations called `showSuccessToast` `showDangerToast` etc, so we only send in `message`, and we don't need to send in `open: true` and `variant="success"`

What do you think?

Reply:
Yeah, that would be great.

Reply:
Cool! I made showDangerToast and showSuccessToast that we can use. 

I keep `showDangerToast` in actions when we throw an error for now. I think it is good to just have that by default, so all API calls at least show some sort of error if something fails. In the future, we could work on some more context-specific error handling on each component/form field.